.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-Please go the the `Preview` tab and select the appropriate sub-template:
+Please go to the `Preview` tab and select the appropriate sub-template:
 
 * [Production code](?expand=1&template=production.md) - for types `fix`, `feat`, and `refactor`.
 * [Docs](?expand=1&template=docs.md) - for documentation changes.
-* [Others](?expand=1&template=others.md) - for changes that do not affect production code.
\ No newline at end of file
+* [Others](?expand=1&template=others.md) - for changes that do not affect production code.
```

Comment:
Why does this line show as modified? I can't spot any difference, and a diffchecker also claims they are the same. Is there something subtle?

Reply:
Hey, it's a bug on the Github web editor. It sometimes does this to the last line of markdown files.

Reply:
Cool, thanks for clarifying.